src/types/modifications/AddTableCols.jl

```diff
@@ -0,0 +1,40 @@
+
+
+struct AddTableCols <: Modification 

```

Comment:
Be sure to add a doc string before we merge it!  A couple of potential changes:
* Rename the mod to `LeftJoinCols` to be a little more clear about what is happening (or `JoinCols`)
* Rename `join_cols` to `on` to match convention for joining
* Rename `table` to `left_table_name`
* Rename `cols_file` to `right_table_file`
* Rename `cols_table` to `right_table`